kstests-in-cloud.sh

```diff
@@ -71,6 +71,8 @@ Options:
                              the file is located in linchpin directory, default is "PinFile"
 
     -k, --key-name NAME      name of the ssh key used for provisioning in cloud;
+                             this applies to an existing (already upleaded) key or a new
+                             key you want to upload (see --public-key-upload)
```

Comment:
It applies to a newly generated key as well - it will be named NAME

Reply:
Fixed.

Reply:
I dndn't mean to use this phrase as it is, I'd go with
"this applies to the name of a newly generated key (default), or to an already existing key that should be used ( --key-use-existing), or to the name under which a key will be uploaded (--public-key-upload)"

Reply:
ohh, that you meant by the first comment

Reply:
Rephrased.

Reply:
I'd change "or to" -> "or" but I'll create a separate PR for that so we can finally merge this PR